docs/getting-started/verify.md

```diff
@@ -32,6 +32,69 @@ The following checks were performed on each of these signatures:
 Getting the above output followed by a JSON array of payloads, ensures you the
 container image was signed using our private key.
 
+## Verify the container image at runtime
+
+You can also verify the container image at runtime by using `policy-controller`.
+To do so, you first need to [install](https://docs.sigstore.dev/policy-controller/installation/) this component on your kubernetes cluster.
+We will then deploy an `admission-controller` using the following YAML file:
+
+```yaml
+apiVersion: policy.sigstore.dev/v1beta1
```

Comment:
What about enabling this by default during deployment (like you're proposing for [gadget images](https://github.com/inspektor-gadget/inspektor-gadget/pull/2678#discussion_r1551061245))?

I mean, by default, we verify if the `policy-controller` is installed in the cluster:
- If it's, we deploy the `admission-controller` and verify the image. 
- If it isn't, we print something like "Warn: The policy-controller wasn't found: Impossible to verify image" or so.

Finally, if users don't want this verification, they use the flag `--verify-image=false`.

Reply:
> I just need to think how to articulate between `kubectl-gadget` and `ig`
>
What do you mean? This is only for verifying the container image for the gadget daemonset. How is this related to `ig`? Did I miss something (again 😅)?

Reply:
All clear now. Thanks for working on this. I had to read about cosing and all signing stuff but it was fun 🙂